docs/EKS.md

```diff
@@ -215,6 +219,22 @@ spec:
         effect: <string> : the effect of the taint
 ```
 
+### PlacementSpec
+
+Represents the EC2 Placement information for your EC2 instances.
+All fields are supported for Launch Templates; Launch Configuration's only support setting `tenancy`.
```

Comment:
Should we also add another example such as the one below to show the usage in case of LaunchConfiguration.